themes/custom/az_barrio/az_barrio.theme

```diff
@@ -182,6 +316,10 @@ function az_barrio_library_info_alter(&$libraries, $extension) {
     $libraries['az-brand-icons']['css']['theme'] = [
       $az_brand_icons_path => $az_brand_icons_css_info,
     ];
+     
+    if (theme_get_setting('use_material_design_sharp_icons') === true) {
+    $libraries['arizona-bootstrap']['dependencies'][] = 'az_barrio/material-design-icons-sharp';
```

Comment:
Nitpick: whitespace

Reply:
General question:  Are the material icons a dependency for the off-canvas nav and search block?  Are they still usable if this setting is disabled?